classes/ActionScheduler_AdminView.php

```diff
@@ -147,6 +147,11 @@ protected function check_pastdue_actions() {
 		$threshold_seconds = ( int ) apply_filters( 'action_scheduler_pastdue_actions_seconds', DAY_IN_SECONDS );
 		$threshhold_min    = ( int ) apply_filters( 'action_scheduler_pastdue_actions_min', 1 );
 
+		# Set fallback value for past-due actions count.
+		if ( ! isset( $num_pastdue_actions ) ) {
```

Comment:
I don't think the conditional is necessary. This close to the start of the method, we know that `$num_pastdue_actions` will not yet be set.